app/controllers/projects_controller.rb

```diff
@@ -14,6 +14,9 @@ def show
     else
       @project = Project.find_by_owner_and_name(params[:user], params[:repo])
     end
+
+    raise ActiveRecord::RecordNotFound if @project.nil?
```

Comment:
We shouldn't need this as `find_by_owner_and_name` should raise that exception if it's not found: https://github.com/andrew/contribulator/blob/master/app/models/project.rb#L28